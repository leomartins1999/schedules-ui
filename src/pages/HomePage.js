function HomePage() {
    return (
        <>
            <h1>Welcome to the Schedules Comparator Application!</h1>
            <p>To add new schedules, access the Schedules page!</p>
            <p>To compare existing schedules, access the Compare page!</p>
        </>
    );
}

export default HomePage;
