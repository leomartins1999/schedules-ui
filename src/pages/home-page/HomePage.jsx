import Card from 'react-bootstrap/Card'

function HomePage() {
    return (
        <div className="p-2">
            <Card>
                <Card.Body>
                    <Card.Title className="text-center"><h1>ISCTE Schedules Comparator App</h1></Card.Title>
                    <Card.Text>
                        This app can be used to compare already generated schedules and sort out which one most benefits the users.
                        <br/> It can be used for:
                        <br/> - Listing already uploaded schedules
                        <br/> - Uploading new schedules
                        <br/> - Consulting the timetable for already uploaded schedules
                        <br/> - Selecting and comparing uploaded schedules
                        <br/> - Executing a comparison selecting the metrics relevant for said comparison
                    </Card.Text>
                </Card.Body>
                <Card.Footer className="text-muted text-center">
                    Project developed for the ADS project of the Master's Degree in Computer Engineering at ISCTE
                    <br/> Â© Leonardo Martins, 2021
                </Card.Footer>
            </Card>
        </div>
    );
}

export default HomePage;
