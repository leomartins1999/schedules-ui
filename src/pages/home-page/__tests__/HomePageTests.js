import renderer from 'react-test-renderer';
import HomePage from '../HomePage'

it('renders the HomePage', () => {
    const tree = renderer
        .create(<HomePage />)
        .toJSON();

    expect(tree).toMatchSnapshot();
})
