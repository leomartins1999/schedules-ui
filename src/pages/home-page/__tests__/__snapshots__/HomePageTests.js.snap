// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the HomePage 1`] = `
<div
  className="p-2"
>
  <div
    className="card"
  >
    <div
      className="card-body"
    >
      <div
        className="text-center card-title h5"
      >
        <h1>
          ISCTE Schedules Comparator App
        </h1>
      </div>
      <p
        className="card-text"
      >
        This app can be used to compare already generated schedules and sort out which one most benefits the users.
        <br />
         It can be used for:
        <br />
         - Listing already uploaded schedules
        <br />
         - Uploading new schedules
        <br />
         - Consulting the timetable for already uploaded schedules
        <br />
         - Selecting and comparing uploaded schedules
        <br />
         - Executing a comparison selecting the metrics relevant for said comparison
      </p>
    </div>
    <div
      className="text-muted text-center card-footer"
    >
      Project developed for the ADS project of the Master's Degree in Computer Engineering at ISCTE
      <br />
       Â© Leonardo Martins, 2021
    </div>
  </div>
</div>
`;
